ui/src/routes/guide.svelte

```diff
@@ -0,0 +1,141 @@
+<script>
+	import LinkButton from 'app/components/LinkButton.svelte';
+	import ExternalLink from 'app/components/ExternalLink.svelte';
+	import {version} from 'app/utils';
+
+	const csvWikiURL = 'https://en.wikipedia.org/wiki/Comma-separated_values';
+</script>
+
+<svelte:head>
+	<title>BEIS indicators - Methodology</title>
+</svelte:head>
+
+<main>
+	<section>
+		<h1>How to explore the indicators</h1>
+
+		<h2>Main page</h2>
+
+		<h3>Temporal ditribution</h3>
+		<p>On the right you find a representation of temporal ditribution or the available indicators.<p>
+		<p>Each circle represents data for an indicator being available for that year.<p>
+		<p>Click on it to the navigate to that indicator for that specific year.<p>
+
+		<h3>Timeline</h3>
+		<p>Here you find a list of all the years where there's data available for at least one indicator.<p>
+		<p>In this page, the timeline is not interactive.<p>
+
+		<h3>Sidebar</h3>
+		<p>Here you find a list of indicators, grouped in various thematic sections.<p>
+		<p>Clicking on a title will navigate to the trend view for that indicator.<p>
+
+		<h2>Indicator trends</h2>
+
+		<h3>Timeline</h3>
+		<p>The Timeline shows one white dot for each available year. It is now interactive in that by clicking on a dot lets you navigate to the indicator yearly view for that year.<p>
+		<p>In this page, the timeline is not interactive.<p>
+
+		<h3>Sidebar</h3>
+		<p>The sidebar has the same function described above: you can use it to navigate to the trends page of the available indicators.<p></p>
+
+		<h3>Trends</h3>
+		<p>Here you find a set of trend lines, one for each selected NUTS2 regions.</p>
+		<p>Hovering trends, highlights the trend and the data point nearest to the pointer.</p>
+		<p>Please use the switch on the top-right corner to choose between showing trends with absolute or ranked values.</p>
+
+		<h3>Regional selection</h3>
+		<p>It is possible to select NUTS1 regions to inherently select its contained NUTS2 region. This is particularly useful because NUTS1 regions don't vary over the years whereas NUTS2 regions do.</p>
```

Comment:
```suggestion
		<p>The trend lines shown correspond to NUTS2 regions, however it is possible to select and filter regions based on the larger NUTS1 regions, of which they are a part. This is particularly useful for clarity when focusing on specific regions and also because NUTS1 regions don't vary over time whereas NUTS2 regions do.</p>
```